docs/src/index.md

```diff
@@ -10,9 +10,7 @@ using CairoMakie, Attractors
 
 ## Latest news
 
-- Our paper on the global stability analysis framework offered by Attractors.jl ([`continuation`](@ref)) and the novel continuation offered by [`RecurrencesFindAndMatch`](@ref) is published as a _Featured Article_ in Chaos (https://pubs.aip.org/aip/cha/article/33/7/073151/2904709/Framework-for-global-stability-analysis-of) and has been featured in the AIP publishing showcase (https://www.growkudos.com/publications/10.1063%25252F5.0159675/reader)
-- New function [`minimal_fatal_shock`](@ref)
-- New function [`match_continuation!`](@ref) which improves the matching during a continuation process where attractors disappear and reappear.
+- New functions [`edgetracking`](@ref) and [`bisect_to_edge`](@ref) added that implement the **edge tracking algorithm**, a method to find saddles, *edge states* or *Melancholia states* in dynamical systems, also when they are unstable chaotic sets.
```

Comment:
```suggestion
- New functions [`edgetracking`](@ref) and [`bisect_to_edge`](@ref) added that implement an **edge tracking algorithm**, a method to find saddles or *edge states* in dynamical systems, also when they are unstable chaotic sets.
```

there are many algorithms, such as the saddle stradle, and this is one of them. (not "the")

(Also, let's stick to commonly accepted terminology and keep the docstring lean, it is fine to refer to this term in the rest of the docs)